### PR TITLE
Add example using Mongo Seeding with Mongoose and Express

### PR DESCRIPTION
**Pull request by D4ITON**
Hi to all, I needed to use a seeder to my project, so I found this helpfull package, it is super, I have created an example using json, one in Spanish and other in English, I hope to help the package.

### CODE REVIEW

**Comment by pkosiec** [comment]
Hi @D4ITON!
Thank you very much for your contribution. Nice work!

I think that's a very good example of using Mongo Seeding along with Mongoose.
While the documentation need some changes to merge it here,  I think your Readme in Spanish or English is ready to use as a blog post. You can publish it on Medium, Dev.to or other platforms. What do you think?

Then you would have the opportunity to use the content as it is. And here, after some modifications, your example will be merged too :-) I will do code review soon!

File: examples/import-data-json/README.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+```
+After to be sure the we're correctly in the route, we use the command:
+```bash
+$ seed -u mongodb://127.0.0.1:27017/restaurant --drop-database ./data
+```
+
+When we execute the command above, it does not give us a message but if you look at your database GUI you will notice that you already have your tables populated with the information that we have placed.
+
+
+### 4. Fetch data with mongoose populate
```
Comment (pkosiec) [inline examples/import-data-json/README.md:191]:
```suggestion
### Fetch data from DB with Mongoose
```

File: examples/import-data-json/README es.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
+![Mongo Seeding](https://raw.githubusercontent.com/D4ITON/mongo-seeding/master/docs/assets/logo-express-mongodb-mongoose-mongo-seeding.png)
```
[the remark this answers is not in the document]
Comment (D4ITON) [reply examples/import-data-json/README es.md]: Thank you very much for the consideration Paweł, I made some improvements reading the points you put, I will be aware of your answer 👍

**Comment by D4ITON** [comment]
> Hi @D4ITON!
> Thank you very much for your contribution. Nice work!
> 
> I think that's a very good example of using Mongo Seeding along with Mongoose.
> While the documentation need some changes to merge it here, I think your Readme in Spanish or English is ready to use as a blog post. You can publish it on Medium, Dev.to or other platforms. What do you think?
> 
> Then you would have the opportunity to use the content as it is. And here, after some modifications, your example will be merged too :-) I will do code review soon!

File: examples/mongoose-express/README.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+After run the command it does not give us a message but if you look at your database GUI you will notice that you already have your tables populated with the information :rocket:
+
+## Fetch data from DB with Mongoose
+To bring the defined relation of customers, menus, and orders you must use the [populate](https://mongoosejs.com/docs/populate.html) function of Mongoose, the function it can be the following:
+
+```js
+const Order = require("./models/order");
+
+api.get("/get-orders", (req, res) => {
+    Order.find()
```
Comment (pkosiec) [inline examples/mongoose-express/README.md:184]: Please create `index.js` file in the `examples/mongoose-express/example` directory and paste this code there. Thanks!

File: examples/mongoose-express/README.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+
+## Use Mongo Seeding CLI
+To use it in more than one project, you must install it globally with this command:
+
+```bash
+npm install -g mongo-seeding-cli
+```
+
+You can specify custom settings with command line parameters. The following example imports data from `./mongoose-express/example/data` directory using MongoDB connection URI mongodb://127.0.0.1:27017/mydb with option to drop database before import:
+> **NOTE:** You can use the [following parameters](../../cli/README.md#command-line-parameters) while using `seed` tool
```
Comment (pkosiec) [inline examples/mongoose-express/README.md:169]:
```suggestion

> **NOTE:** You can use the [following parameters](../../cli/README.md#command-line-parameters) while using `seed` tool
```

File: examples/mongoose-express/README.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+
+```bash
+seed -u mongodb://127.0.0.1:27017/dbname --drop-database ./data
+```
+
+After run the command it does not give us a message but if you look at your database GUI you will notice that you already have your tables populated with the information :rocket:
```
Comment (pkosiec) [inline examples/mongoose-express/README.md:175]:
```suggestion
Once the database seed finishes, the database is populated with collections from `./data` directory. 
```

Comment (pkosiec) [reply examples/mongoose-express/README.md:175]: BTW:
> After run the command it does not give us a message

Hmm, wait... it shouldn't be like that. Mongo Seeding CLI should output detailed info what's going on... Did you discover a bug? 🤔

Comment (pkosiec) [reply examples/mongoose-express/README.md:175]: Yes you did. Thanks a lot! :smile:

https://github.com/pkosiec/mongo-seeding/issues/114

File: examples/mongoose-express/README.md (hunk 4 above)
Comment (D4ITON) [reply examples/mongoose-express/README.md:169]: > Hi again @D4ITON,
> Awesome work. Almost there! Second (and final) round of comments.
> Thank you so much for your patience! After applying these comments we will be ready to merge it 🙂

Hi, @pkosiec, thanks, dont worry if is need to make changes let's do it :)

File: examples/mongoose-express/README.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+        .exec((err, order) => {
+            if (err) return handleError(err);
+            res.status(200).send(order);
+        });
+})
+```
+
+To test the API, you can run the following curl:
+```bash
+curl --location --request GET 'http://localhost:3978/get-orders'
```
[the remark this answers is not in the document]
Comment (D4ITON) [reply examples/mongoose-express/README.md:195]: In this part change, but bellow is different the line 🤔

File: examples/mongoose-express/README.md [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+    date: Date,
+});
+
+module.exports = mongoose.model("Order", OrderSchema);
+```
+
+## Define proper directory structure
+To learn how to structure import data, see the [Import data definition guide](../../docs/import-data-definition.md). In this example the following directory structure is used:
+
+> **NOTE:** It is important that the end of the folder name (collection name) ends in plural.
```
[the remark this answers is not in the document]
Comment (D4ITON) [reply examples/mongoose-express/README.md:70]: mm It's good, I hadn't noticed 😄

File: examples/mongoose-express/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (D4ITON) [reply examples/mongoose-express/README.md:195]: I pasted bellow to test the app

[the remark this answers is not in the document]
Comment (pkosiec) [reply examples/mongoose-express/README.md]: Yup, it's different, but it works, because `curl` by default does GET request 🙂

**Comment by pkosiec** [comment]
@D4ITON Thanks for the improvements and patience.

I gave the final touch to the document - just some minor things 🙂 I didn't want to bother you with another comments. Anyway, of course at the end there will be one commit on master, with you as an author. All credit goes to you, obviously.

Again, thank you so much for your contribution. Cheers!

**Comment by D4ITON** [comment]
So much thanks to you @pkosiec , I just wanted to leave my grain of sand 🎉, I'm so happy to had do it 😄 Cheers!

**Pull request merged**